lib/airbrake/worker.ex

```diff
@@ -131,8 +139,8 @@ defmodule Airbrake.Worker do
 
   defp ignore?(nil, _type, _message), do: false
   defp ignore?(:all, _type, _message), do: true
-  defp ignore?(%MapSet{} = types, type, _message), do: MapSet.member?(types, type)
   defp ignore?(fun, type, message) when is_function(fun), do: fun.(type, message)
+  defp ignore?(types, type, _message), do: MapSet.member?(types, type)
```

Comment:
`MapSet` is an opaque type, and apparently pattern matching on it offends dialyzer.  So, make this the default clause since all others have pattern matching or a guard.